mmdet/models/roi_heads/mask_heads/fcn_mask_head.py

```diff
@@ -159,12 +177,38 @@ def get_seg_masks(self, mask_pred, det_bboxes, det_labels, rcnn_test_cfg,
                 it will be converted to numpy array outside of this method.
             det_bboxes (Tensor): shape (n, 4/5)
             det_labels (Tensor): shape (n, )
-            img_shape (Tensor): shape (3, )
             rcnn_test_cfg (dict): rcnn testing config
-            ori_shape: original image size
+            ori_shape (Tuple): original image height and width, shape (2,)
```

Comment:
Two arguments `scale_factor` and `rescale` are missing.

Reply:
I find these two arguments confusing. It seems out of the scope of this PR to add those, but I gave it a shot. Please directly edit the docs or make a suggested edit (I believe maintainers can directly edit the branch) if they aren't to your liking.

I wrote a very literal description of what they are doing because I don't understand the intuition for when they should be used.